YAMP.Core/Operators/BinaryOperator.cs

```diff
@@ -48,8 +48,20 @@ public virtual Value Handle(Expression left, Expression right, IDictionary<Strin
 			return Perform(l, r);
 		}
 
-        internal Value PerformOverFind(Value left, Value right, BinaryOperatorMappingList mapping)
+        public Value PerformOverFind(Value left, Value right, BinaryOperatorMappingList mapping)
         {
+            Boolean found;
+            var ret = TryPerformOverFind(left, right, mapping, out found);
+            if (!found)
+                throw new YAMPOperationInvalidException(Op, left, right);
+
+            return ret;
+        }
+
+        public static Value TryPerformOverFind(Value left, Value right, BinaryOperatorMappingList mapping, out Boolean found)
```

Comment:
Not really.
I just "extended" the original PerformOverFind with the "extra" output, instead of throwing the YAMPOperationInvalidException, but minimizing the code changes inside refactored code inside the function.
It didn't seemed strange at that time, but I agree with normalization of the Try pattern.
Do you want me to submit the changes?
The function is being called in a few places, which will have to change also, but it's an easy fix.